src/workerskv/list_namespace_keys.rs

```diff
@@ -7,6 +7,7 @@ use crate::endpoint::{Endpoint, Method};
 pub struct ListNamespaceKeys<'a> {
     pub account_identifier: &'a str,
     pub namespace_identifier: &'a str,
+    pub params: ListNamespaceKeysParams,
```

Comment:
Why not `&ListNamespaceKeysParams`?

Reply:
that's a great question. the suggested code i received (and merged) for the CreateNamespace endpoint did not use a reference for params. I can add an issue and PR to fix that in other places.

Reply:
Awesome!